build/config/android/rules.gni

```diff
@@ -1753,6 +1753,47 @@ template("android_apk") {
     }
   }
 
+  if (defined(invoker.flutter_dist_jar)) {
```

Comment:
Do we need to integrate this tightly with android_apk ?  It seems strange for android_apk, which is a very general-purpose rule, to know about Flutter.